selfdrive/ui/qt/offroad/developer_panel.cc

```diff
@@ -111,6 +116,7 @@ void DeveloperPanel::updateToggles(bool _offroad) {
     hyundaiRadarTracksToggle->setVisible(false);
   }
   experimentalLongitudinalToggle->refresh();
+  neuralNetworkLateralControl.refresh();
```

Comment:
if not SP, this might blow up. Do we really need the if sunnypilot above? (likely the answer is yes, but still had to ask)

Reply:
nvm, yeah, the only way it wouldn't be with that IF is if we make our own sunnypilot_developer panel and inherit and yada yada yada

Reply:
We'll have a dedicated panel for this. `Developer` is a temporary home for them :)